tests/wpt/setters_tests.json

```diff
@@ -110,7 +110,7 @@
             }
         },
         {
-            "comment": "Can’t switch from URL containing username/password/port to file",
+            "comment": "Can't switch from URL containing username/password/port to file",
```

Comment:
We shouldn't change this file. If we do, the next WPT updater script will override it.

Reply:
fixed